wazo_install.sh

```diff
@@ -74,12 +74,55 @@ install_wazo () {
     fi
 }
 
+get_ansible_installer() {
+    # Install git
+    apt-get update
+    apt-get install -yq git curl
+
+    # Setup Ansible directory
+    rm -rf /usr/src/wazo-ansible
+    cd /usr/src
+
+    # Get Ansible installer
+    git clone https://github.com/wazo-pbx/wazo-ansible 
+    cd wazo-ansible
+
+    # Find the last tag if requested
+    if [ $tag = LAST ]; then
+        tag=$(curl https://mirror.wazo.community/version/stable)
+    fi
+
+    git checkout $tag
```

Comment:
```suggestion
    git checkout wazo-$tag
```
* the tag `wazo-19.04` is missing from `wazo-ansible` repository
* This mechanic is not really compatible with backport (e.i. tag `wazo-19.06.1`)

Reply:
* I added `wazo-19.04` in `wazo-pbx/wazo-ansible`
* What do you think about force-pushing `wazo-19.06` on `wazo-19.06.1` and adding `wazo-19.06.0` in the original `wazo-19.06` commit?

Reply:
I don't have a strong preference, but it seems a little bit hackish. What would we do if we have tag 19.06.2 ? Also is not symmetric with other tag for other repository

Reply:
Well it would need a change in the hotfixing procedure (which does not exist yet, really): Each time we hotfix a version, tag the previous version with `XX.XX.0` and force-move the `XX.XX` tag to the latest hotfixed commit.

Another solution would be to look for the latest tag beginning with `wazo-$tag` and `git checkout` that tag, something like:
```
tag=$(curl https://mirror.wazo.community/version/stable)
latest_tag=$(git tag --list --sort version:refname "wazo-$tag*" | tail -n 1)
```
The `--sort version:refname` avoids listing `wazo-19.06.10 before `wazo-19.06.2`.

Reply:
I prefer the second option, since moving a git tag seems weird to me (and what it can be if we have 2 hotfix version)

Reply:
It's the same logic than the Docker tags :)

Reply:
ok well then I don't care :laughing: